src/components/wallet/AddWalletWidget/CustomWalletForm/CustomWalletForm.jsx

```diff
@@ -9,7 +9,7 @@ import { Map } from 'immutable'
 import React, { PureComponent } from 'react'
 import { connect } from 'react-redux'
 import { Translate } from 'react-redux-i18n'
-import { TextField } from 'redux-form-material-ui'
+import { TextField } from '@material-ui/core'
```

Comment:
Could you please also check another places with possibly wrong imports of TextField from 'redux-form-material-ui' instead of '@material-ui/core'?